### PR TITLE
IODEMO/LOG: added logging of handles signals

### PR DESCRIPTION
**Pull request by hoopoepg**
- added logging of handled signals to simplify logs parsing

### CODE REVIEW

File: test/apps/iodemo/io_demo.cc [hunk 1]
```diff
@@ -682,6 +719,12 @@ class P2pDemoCommon : public UcxContext {
 
     static void signal_terminate_handler(int signo)
     {
+        if ((unsigned)signo < ucs_static_array_size(signame)) {
+            LOG << "Run-time signal handling: " << signame[signo];
```
Comment (yosefe) [inline test/apps/iodemo/io_demo.cc:723]: IMO, printing just signal number is enough

Comment (hoopoepg) [reply test/apps/iodemo/io_demo.cc:723]: just ```SIGINT```?
may be add any word to simplify grep?

Comment (yosefe) [reply test/apps/iodemo/io_demo.cc:723]: I'd just print the number (2), prefer to not add all these constants to io_demo
Or, maybe Linux/glibc has some method to get signal name from number

**Review by yosefe — APPROVED** [review]

File: test/apps/iodemo/io_demo.cc (hunk 1 above)
Comment (hoopoepg) [reply test/apps/iodemo/io_demo.cc]: done

**Comment by yosefe** [comment]
@hoopoepg can u pls squash?

**Comment by hoopoepg** [comment]
@yosefe ok to merge?
back it port into int3?

**Comment by yosefe** [comment]
> back it port into int3?

@hoopoepg yes pls

**Pull request merged**